FirstProject/css/feedback.css

```diff
@@ -0,0 +1,162 @@
+/*Feedback HTML*/
+
+.footer_section {
```

Comment:
Excellent! The reason why we're trying to instill this mentality early on is that this helps overall maintenance greatly. You'll only ever need to touch a single piece of code to affect the look and feel of your component(s) throughout the app and you won't be forced to keep track of _all_ the instances where you've included styling for the footer. Such an approach is prone to human error, especially on large projects / in large teams, where not everyone is aware of absolutely everything ever developed. :)